### DISCUSSION TITLE
I'm trying to see a commit but it says "This page is taking too long to load"

### DISCUSSION THREAD

**Discussion by maurocatania** [community/community#40926] [Other Feature Feedback, Questions, & Ideas] [answered]
I have tried many times but I can't get it to load. The commit I'm trying to see is: https://github.com/mosh-hamedani/vidly-mvc-5/commit/b727a26e1b4b88abe84a8b48208fec537db2ed43

Is there a way to fix this? Thank you 

![Screenshot_1](https://user-images.githubusercontent.com/106556463/205926407-dda7d14e-2804-4f54-becf-bccc886cb7aa.png)

**Comment by mohamedzhioua** [comment, marked as answer]
maybe it's your internet cnx 
you can use sourcetree "https://www.sourcetreeapp.com/"

**Reply by maurocatania** [reply]
Thank you!! It worked :)
